haas/test_common.py

```diff
@@ -45,20 +45,36 @@ def releaseDB(db):
     pass
 
 
+def config_clear():
+    """Clear the contents of the current HaaS configuration"""
+    for section in cfg.sections():
+        cfg.remove_section(section)
+
+
+def config_set(config_dict):
+    """Set the configuration according to ``config_dict``.
+
+    ``config_dict`` should be a dictionary mapping section names (strings)
+    to dictionaries mapping option names within a section (again, strings)
+    to their values.
+    """
+    config_clear()
+    for section in config_dict.keys():
+        cfg.add_section(section)
+        for option in config_dict[section].keys():
+            cfg.set(section, option, config_dict[section][option])
+
```

Comment:
Ack, it totally should be shared. I probably wrote this one first and forgot about it --- just dealt with the merge on another branch, now I know why it was so big...

This one isn't going into master anyway; if we decide what to do with #437, I can merge it back into this.